packages/build/webpack/package.config.js

```diff
@@ -7,7 +7,6 @@ const path = require('path');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const constants = require('./constants');
 const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
-const IgnorePlugin = require('webpack').IgnorePlugin;
```

Comment:
> We have had so many edge case type date related issues. I think we've done a pretty good job of adding jest or selenium tests for these as we fix them, but do you think there is a need for a manual testing pass on this fb as well for those issues called out (mostly in util/Date.ts)?

As discussed, I've audited the regression coverage for all issues specified in `util/Date.ts` and am looking to ensure full coverage. Here is the audit results:

- [x] [Issue 44011: account for the shortcut values (i.e. "Date", "DateTime", and "Time")](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=44011)
  - Coverage: Date.test.ts `getColDateFormat.'shortcut formats'`
- [x] [Issue 48300: Respect 12-hour vs 24-hour time display format](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=48300)
  - Coverage: Date.test.ts `parseDateFNSTimeFormat.'various formats'`
- [x] [Issue 46460: Allow searching by date only (not time)](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=46460)
  - Coverage: Date.test.ts `getColFormattedDateFilterValue`
- [x] [Issue 48608: DateTime format to include timezone results in unhandled client side error](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=48608)
  - Coverage: Date.test.ts `getColDateFormat.'date-fns replacement'`
- [ ] [Issue 30834: Respect Date Format Settings in LabKey Biologics](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=30834)
  - Coverage: NEEDS COVERAGE
- [ ] [Issue 45140: Date picker will not always populate a date time field when format has day first (dd/MM/yyyy) instead of month first (MM/dd/yyyy)](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=45140)
  - Coverage: (PARTIAL COVERAGE) Date.test.ts `parseDate`. Need more coverage of `dateOnly` and `timeOnly` parameters (I don't think these parameters are actually related to this issue but branch coverage is not there so I'm saying it is).
- [x] [Issue 46767: Using the editable grid doesn't enforce date values then causes problems](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=46767)
  - Coverage: Date.test.ts `parseDate.minDate`
- [x] [Issue 44398: LKSM: Date/time picker passing PST value instead of EST ](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=44398)
  - Coverage: Date.test.ts `getJsonDateTimeFormatString`

Reply:
@cnathe This is ready for your re-review. Thanks.